docs/how-tos/how-to-reset-state.mdx

```diff
@@ -34,3 +34,36 @@ const reset = () => {
   state.obj = _.cloneDeep(initialObj)
```

Comment:
I mean this is wrong too. L31-35

Can you please fix the doc in this way?
- Fix cloneDeep usage with v2.
- Add a note about structuredClone

Speaking of v2, we now have `deepClone` from valtio/utils, we don't need to use cloneDeep.
So, we could restructure the docs to use `deepClone` first and `structuredClone` second.
(I thought about explaining structuredClone first, but `deepClone` is capable of avoid copying `ref()`'d object.

Reply:
Sure thing. I'll push up the changes soon.

Reply:
Changes have been pushed. Let me know of any feedback.